packages/clerk-auth/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@hono/clerk-auth",
+  "version": "0.0.0",
+  "description": "A third-party Clerk auth middleware for Hono",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "test": "jest",
+    "build": "rimraf dist && tsc",
+    "prerelease": "yarn build && yarn test",
+    "release": "yarn publish"
+  },
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org",
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/honojs/middleware.git"
+  },
+  "homepage": "https://github.com/honojs/middleware",
+  "peerDependencies": {
+    "hono": "*"
+  },
+  "devDependencies": {
+    "hono": "^2.7.2"
+  },
+  "dependencies": {
+    "@clerk/backend": "^0.28.1"
```

Comment:
Perhaps, it might be better to make `@clerk/backend` as `peerDependencies`. And fix the install command as the following:

```plain
npm i hono @hono/clerk-auth @clerk/backend
```

Reply:
Hello @yusukebe :) Was going through this PR and I'd like to ask if there were any specific reasons you prefer `npm i hono @hono/clerk-auth @clerk/backend` instead of `npm i hono @hono/clerk-auth` and making `@clerk/backend` a direct dependency?

Please feel free to ignore my question completely as I'm only asking out of curiosity. Thank you!

Reply:
Hi @nikosdouvlis 

I didn't have any significant reasons. This is better for importing `@clerk/backend` in other places but does not seem necessary.